Application/src/Screens/ConnexionScreen.tsx

```diff
@@ -22,8 +22,24 @@ export default function ConnexionScreen() {
         console.log("Act on forgot password")
     }
 
-    function connectionAction() {
+    async function connectionAction() {
         console.log("Connect user", userMail, userPass)
+
+        const requestOptions = {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({email: userMail, password: userPass})
+        }
+
+        try {
+            await fetch("http://10.29.125.228:8080/login", requestOptions).then(response => {
```

Comment:
Why this IP address ?

Reply:
the ip of the server. At the moment it's my PC, as we don't host the server. This will change when we migrate to Azure.
If you want to I can require a configuration file for the ip which will be in the .gitignore and which will be different for all members of the group.